content/md/imports-exports/tailored-export.md

```diff
@@ -100,8 +101,22 @@ Limitations have been set for each Tailored Export profile:
 You can use operations to share the most meaningful values for your stakeholders by adapting your PIM values in different ways when exporting them.
 
 Here are the different types of operations you can add when defining your sources:
-* `Boolean replacement`: Replace the default values of the boolean attribute with your own value. For now, this is only available for the `Enabled` system property
-* `Concatenate`: Concatenate different sources by displaying up to 4 of them per column
-* `Select labels`: Export the labels of your sources instead of the codes
-
+* `Boolean replacement`: Replace the default values of the boolean attribute with your own value. For now, this is only available for the `Enabled` system property.
+* `Concatenate`: Concatenate different sources by displaying up to 4 of them per column. <!-- Click on `Add text` to define a text separator of your choice.-->
+* `Select labels`: Export the labels of your sources instead of the codes.
+* `Use default value when empty`: Define a default value when the source value is empty.
```

Comment:
Should we need to specified that it's not available on parent, categories and association source ?